Docs build is failing

<!--
Welcome! Before creating a new issue:
* Search for relevant issues
* Follow the issue reporting guidelines:
https://jupyterlab.readthedocs.io/en/latest/getting_started/issue.html
-->

## Description
Docs builds are faling with:

```
=================================== FAILURES ===================================
_ /home/runner/work/jupyterlab/jupyterlab/docs/build/html/developer/contributing.html: https://jupyter.readthedocs.io/en/latest/contributor/content-contributor.html _
https://jupyter.readthedocs.io/en/latest/contributor/content-contributor.html: 404: Not Found
```

example: https://github.com/jupyterlab/jupyterlab/runs/860658977

--- a/docs/source/developer/contributing.rst
+++ b/docs/source/developer/contributing.rst
@@ -33,7 +33,7 @@ General Guidelines for Contributing
 
 For general documentation about contributing to Jupyter projects, see
 the `Project Jupyter Contributor
-Documentation <https://jupyter.readthedocs.io/en/latest/contributor/content-contributor.html>`__
+Documentation <https://jupyter.readthedocs.io/en/latest/contributing/content-contributor.html>`__
 and `Code of
 Conduct <https://github.com/jupyter/governance/blob/master/conduct/code_of_conduct.md>`__.
 

